### ISSUE TITLE
Type loss in inputs.prometheus => outputs.prometheus_client

### ISSUE THREAD

**Issue by Boiler** [influxdata/telegraf#3888]
## Bug report

Loading the data with telegraf inputs.prometheus and exporting it with the prometheus output

- Type becomes untyped (# TYPE log_lines untyped)

### Relevant telegraf.conf:

```
[[outputs.prometheus_client]]
  listen = "127.0.0.1:9126"

[[inputs.prometheus]]
  urls = ["http://localhost:3903/metrics"]
```

### System info:

Telegraf v1.5.2 (git: release-1.5 67440c95)
CentOS Linux release 7.4.1708 (Core)

### Steps to reproduce:

```
$ curl -s localhost:3903/metrics
# TYPE log_lines counter
log_lines{level="I",instance="srv1"} 598494
log_lines{level="E",instance="srv1"} 6944
log_lines{level="W",instance="srv1"} 53
```

### Expected behavior:

```
$ curl -s localhost:9126/metrics
# HELP log_lines counter Telegraf collected metric
# TYPE log_lines counter
log_lines{instance="srv1",level="E",url="http://localhost:3903/metrics"} 6944
log_lines{instance="srv1",level="I",url="http://localhost:3903/metrics"} 598494
log_lines{instance="srv1",level="W",url="http://localhost:3903/metrics"} 53
```

### Actual behavior:

```
$ curl -s localhost:9126/metrics
# HELP log_lines_counter Telegraf collected metric
# TYPE log_lines_counter untyped
log_lines_counter{instance="srv1",level="E",url="http://localhost:3903/metrics"} 6944
log_lines_counter{instance="srv1",level="I",url="http://localhost:3903/metrics"} 598494
log_lines_counter{instance="srv1",level="W",url="http://localhost:3903/metrics"} 53
```

### Additional info:

Very similar issue: https://github.com/influxdata/telegraf/issues/3602

**Comment by jdoupe** [comment]
This was actually fixed back in 1.5 (https://github.com/influxdata/telegraf/pull/3351). Not sure where the regression is affecting it.

**Comment by danielnelson** [comment]
I believe you are experiencing a side effect of #4220, this is fixed in 1.6 and newer.
